recipes/server.rb

```diff
@@ -99,7 +99,17 @@
 end
 
 bash 'openvpn-initca' do
-  environment('KEY_CN' => "#{node['openvpn']['key']['org']} CA")
```

Comment:
Any reason why some of these are `bash` resources and others are `execute`? Might be good to switch them all to `execute` for consistency.

Reply:
Not as far as I can tell, I'll change that.